From 5784ca89309859b67d645bc3781ed3fbfb02d3dd Mon Sep 17 00:00:00 2001
From: Lee Shombert <shombert@google.com>
Date: Fri, 15 Aug 2025 12:05:48 -0700
Subject: [PATCH 1/2] Select sqlite 3.44.5 on ap3a

This selects sqlite 3.44.5 on ap3a and later releases.  This does not
affect 25Q4, which explicitly selects a newer sqlite.

Tested by booting an image on Akita and verifying that the sqlite
version is 3.44.5.

Flag: RELEASE_PACKAGE_LIBSQLITE3
Bug: 430889718
Bug: 281064726
Test: presubmit
Change-Id: I6ba60fa46a775cadbf9f9d4e236b8e550bbd4735
---
 flag_values/ap3a/RELEASE_PACKAGE_LIBSQLITE3.textproto | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/flag_values/ap3a/RELEASE_PACKAGE_LIBSQLITE3.textproto b/flag_values/ap3a/RELEASE_PACKAGE_LIBSQLITE3.textproto
index bb903b8dd..f189f6429 100644
--- a/flag_values/ap3a/RELEASE_PACKAGE_LIBSQLITE3.textproto
+++ b/flag_values/ap3a/RELEASE_PACKAGE_LIBSQLITE3.textproto
@@ -1,4 +1,4 @@
 name: "RELEASE_PACKAGE_LIBSQLITE3"
 value: {
-  string_value: "3440300"
+  string_value: "3440500"
 }
-- 
2.53.0

